src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,61 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+import { getTabById } from '../../helpers/getTabById';
+
+type TabsProps = {
+  tabs: {
+    id: string;
+    title: string;
+    content: string;
+  } [];
```

Comment:
```suggestion
  tabs: Tab[];
```
You have corresponding type